packages/sdk/test/entities/BorrowingVault.test.ts

```diff
@@ -8,14 +8,17 @@ import { Address, BorrowingVault } from '../../src/entities';
 import { ChainId, RouterAction } from '../../src/enums';
 import {
   BorrowParams,
-  ConfigParams,
+  ChainConfigParams,
   DepositParams,
   PermitParams,
   RouterActionParams,
 } from '../../src/types';
 
-const config: ConfigParams = {
+const config: ChainConfigParams = {
   infuraId: process.env.INFURA_ID ?? '',
+  alchemy: {
```

Comment:
alchemy should be also passed to sdk when constructing the classe ?

Reply:
done